hugo/content/blog/2021-03-20-weblate-translation/index.md

```diff
@@ -0,0 +1,75 @@
+
+Making systematic changes always results in foreseen and unforeseen effort, so not only did we have to evaluate whether there were better alternatives around, but also whether it was worth it to invest into switching to it. A migration is not only a change for our integration and hosting process, but also for existing translators that have to potentially learn new processes and interfaces. We had to evaluate not only technical aspects but also terms of service, and human aspects.
+
+When looking for alternatives we quickly came to three big contenders:
+
+1. Transifex - alread established and integrated
+2. Crowdin - a promising alternative
+3. Weblate - official hosted instance of the Weblate open source software
+
+We documented our evaluation and also asked for feedback [in a discussion topic *(#4641)*](https://github.com/mumble-voip/mumble/discussions/4641).
+
+All three provide free hosting for open source (some limits apply). As a project with a small team and very limited resources we are very greatful for this. Without these we would be in a much worse position.
+
+Various useful features are shared between the platforms, while the user interface and user experience differs greatly between them, as well as the technology behind it and resulting way to integrate them.
+
+In the end we decided to initiate a move to [Hosted Weblate](https://hosted.weblate.org/). Apart from the feature aspects we are also glad to move to a hosted open source software for translations.
+
+## Migration to Weblate
+
+The migration itself has mostly technical aspects to it, and a lot of related work. Deep-diving into the localization setup and aspects of our project also showed some inconsistencies and questions for the future, and potential for cleanup which we applied. The migration is/was tracked in [ticket #4727](https://github.com/mumble-voip/mumble/issues/4727).
+
+Weblate provides integration to GitHub and pull request creation. Some advanced configuration scenarios allow for additional customization. Thanks to the integration we were able to phase out our integration and PR creation bot.
+
+Interestingly enough even before finishing the migration and even with notices on Hosted Weblate to not translate yet we received contributions on there.
+
+We have been using Weblate for several weeks now, and have received numerous contributions. Thank you to all existing and future translators!
+
+The process is not as smooth as we would like it to be yet, but we are glad we made the transition already. And we hope existing and future translators like the Hosted Weblate platform.
+
+## Weblate Translation Platform
+
+[Weblate](https://weblate.org/) is Open Source software. The hosted instance services not only us as FOSS but [other projects](https://hosted.weblate.org/projects/) as well.

```

Comment:
```suggestion
[Weblate](https://weblate.org/) is open source software. The hosted instance services not only us as FOSS but [other projects](https://hosted.weblate.org/projects/) as well.

```

Reply:
Didn’t we discuss this in some meeting? I thought we agreed to Open Source. But I guess it is lower case then?

Reply:
We have it as Open Source in the subheadline and open source in the About text on the landing page 🤔 😕